### ISSUE TITLE
FEATURE - Create screenshots for App Store Connect

### ISSUE THREAD

**Issue by SuzGupta** [mikaelacaron/Basic-Car-Maintenance#125]
These, along with the app icon and the eventual ratings and reviews, drive the app's discoverability. It would be great if one of the UX designers contributing here can take this issue. I can expand this with some useful how-to links and examples. We may want a preview video as well, although it's arguable whether that improves App Store Optimization (ASO) / discoverability.

**Comment by mikaelacaron** [comment]
Alternative app icons is covered in #17 !

**Comment by Yashraj49** [comment]
hello @mikaelacaron how about using the below idea a summary about the project.  can It be implemented or used in readme file ?? or you got any other idea about it ??

<img width="1274" alt="Screenshot 2023-10-10 at 7 44 59 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/29746f3d-9557-432a-9f28-d3e4469702a0">

**Comment by mikaelacaron** [comment]
@Yashraj49 I think this would be awesome for the README  and / or the DocC documentation too!

**Comment by Yashraj49** [comment]
ok then I can take this one but need to wait once the project is completed 80-100%

**Comment by mikaelacaron** [comment]
Yup that's what I was thinking too, I'd wait until the last week of October to do this one. I'll assign it to you, but you can still work on another issue if you'd like to for Hacktoberfest.  Because we have to wait on this one

**Comment by Yashraj49** [comment]
ya sure no problem

**Comment by mikaelacaron** [comment]
@Yashraj49 
Thanks for wanting to contribute! Hacktoberfest ended on October 31, if you can still finish this issue by Saturday November 4, I’ll accept it, but otherwise after that I’ll be un-assigning all issues and cleaning up the code to ship this app to the App Store

**Comment by Yashraj49** [comment]
hey @mikaelacaron I have completed some of those . you can have. a look 

![Chart](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/d5bc6d60-2efa-4b3c-8e64-ab2255f299ab)


I am confused what to insert at C1 and C5 headline . if there are any app feature you think that could suit there let me know.

**Comment by mikaelacaron** [comment]
"Hello there" could be "Basic Car Maintenance" instead of at the bottom left
and then the bottom left the big text could be "mange your maintenance events"

C1 could be something about how it's metric and imperial compatible

C5 could be "multiple vehicles" about how you can manage multiple vehicles

Also, the Sign in with Apple is stretched too tall

**Comment by Yashraj49** [comment]
<img width="994" alt="Screenshot 2023-11-01 at 10 40 29 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/87fe98a5-0595-41a3-8536-136ef33a12eb">

@mikaelacaron  how about this ??

**Comment by mikaelacaron** [comment]
Yes!! One small thing change "manage you maintenance events" with only "Manage Maintenance Events"

**Comment by Yashraj49** [comment]
Done @mikaelacaron 

<img width="1007" alt="Screenshot 2023-11-01 at 11 03 47 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/b02958f1-c74f-48ff-8cf6-a93b458ff593">

**Comment by mikaelacaron** [comment]
I like it! Did you make this in Figma? If so DM me on social media your email, and I can add you to the official board
Also for widgets hmm do we want to change the background to just that light gray and you can make up some widget design?

**Comment by SuzGupta** [comment]
Noting: if we are using the default text for Sign up with Apple, we do not need an asset for this; they provide it. See https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/

**Comment by Yashraj49** [comment]
> I like it! Did you make this in Figma? If so DM me on social media your email, and I can add you to the official board Also for widgets hmm do we want to change the background to just that light gray and you can make up some widget design?

no , I didn't make it on Figma there is a open-source  Xcode project (its like a tool moreover ) created using SwiftUI. I am not sure where I found it maybe it was twitter .

talking about the widget can you share me a image how widget looks like when it is on home screen if you have ?? or any image resembling it.

**Comment by mikaelacaron** [comment]
The widget hasn't been created yet, I'd just leave that off for now. If you find that tool, let me know! I'd love to check it out!
Thanks for this graphic! Looks amazing!
Once you send the final one, I'll close this issue

**Comment by Yashraj49** [comment]
> The widget hasn't been created yet, I'd just leave that off for now. If you find that tool, let me know! I'd love to check it out! Thanks for this graphic! Looks amazing! Once you send the final one, I'll close this issue

I can complete it when widget is created , so don't close this issue till widget is created . sure I will let you know about the tool.

**Comment by mikaelacaron** [comment]
Honestly go ahead and just leave the widget off, I'll be submitting the app to the App Store before the widgets are done

**Comment by Yashraj49** [comment]
> Honestly go ahead and just leave the widget off, I'll be submitting the app to the App Store before the widgets are done

so should i remove the widget stack there ?? if so what should i replace with ?

**Comment by mikaelacaron** [comment]
yeah remove the widgets, and does it need to be replaced?

**Comment by Yashraj49** [comment]
> yeah remove the widgets, and does it need to be replaced?

<img width="1042" alt="SummaryView" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/fd231f0f-d102-4a08-b9d6-7fb5f7e5d971">

@mikaelacaron have a look I have removed widget stack.

**Comment by mikaelacaron** [comment]
One small thing, because I am from the US, can you please set your simulator to US, and redo the screenshots, because the date formatting isn't correct for US styled in the two screenshots of the app, and when showing Odometer readings, show how it's mi/km but for two separate cars, rather than the same one

Also what's the name of this tool? Is there any way I can get the source code to how you made this?

**Comment by Yashraj49** [comment]
> One small thing, because I am from the US, can you please set your simulator to US, and redo the screenshots, because the date formatting isn't correct for US styled in the two screenshots of the app, and when showing Odometer readings, show how it's mi/km but for two separate cars, rather than the same one
> 
> Also what's the name of this tool? Is there any way I can get the source code to how you made this?



> yeah remove the widgets, and does it need to be replaced?



> One small thing, because I am from the US, can you please set your simulator to US, and redo the screenshots, because the date formatting isn't correct for US styled in the two screenshots of the app, and when showing Odometer readings, show how it's mi/km but for two separate cars, rather than the same one
> 
> Also what's the name of this tool? Is there any way I can get the source code to how you made this?

here it is 

![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2023-11-02 at 22 12 43](https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/88454884/959f4024-dc5f-4a63-9a43-d719c971801b)

talking about this tool I do have a Xcode project file of it.

**Comment by mikaelacaron** [comment]
Great thanks! I'll close this issue, but technically this graphic is more of a documentation kind of thing than the original issue describes App Store screenshots that would be submitted for App Store Connect 

> talking about this tool I do have a Xcode project file of it.

can you put this in a repo and share it with me?
also if there's an Xcode project, I'm guessing it shows whatever package you're using to make this graphic?

**Comment by Yashraj49** [comment]
> Great thanks! I'll close this issue, but technically this graphic is more of a documentation kind of thing than the original issue describes App Store screenshots that would be submitted for App Store Connect
> 
> > talking about this tool I do have a Xcode project file of it.
> 
> can you put this in a repo and share it with me? also if there's an Xcode project, I'm guessing it shows whatever package you're using to make this graphic?

thanks , are you planning to use that stack design in readme file ?? 
I will share you the original repo of that tool once I find it out , the thing is I found that exactly 2 months ago on my school computer so I need 1-2 days to find it out 

I will share it with you on your social media once I get it .  tell me on what social media you are active so I can share it .

**Comment by mikaelacaron** [comment]
I'm active on everything, DM me on any of them https://mikaelacaron.com/links

> thanks , are you planning to use that stack design in readme file ??

yeah
